app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -221,7 +221,7 @@ class DescriptionEditFragment : Fragment() {
         binding.fragmentDescriptionEditView.updateInfoText()
 
         binding.fragmentDescriptionEditView.isSuggestionButtonEnabled =
-                SuggestedArticleDescriptionsDialog.availableLanguages.contains(pageTitle.wikiSite.languageCode) &&
+                AccountUtil.isLoggedIn && SuggestedArticleDescriptionsDialog.availableLanguages.contains(pageTitle.wikiSite.languageCode) &&
```

Comment:
Couldn't all these conditions be replaced with `MachineGeneratedArticleDescriptionsAnalyticsHelper.isUserInExperiment`?